docker-hub/access-tokens.md

```diff
@@ -51,7 +60,7 @@ You can rename, deactivate, or delete a token as needed.
 the far right of a token row to bring up the edit screen. You can also
 select multiple tokens to delete them all once.
 
-      ![](images/hub-edit-token.png)
+      ![](images/Generate-PAT.png)
```

Comment:
perhaps use the existing naming conventions so that the files can be easily found together (e.g. `hub-generate-token.png` or  `hub-copy-token.png`)

In general; try to use lowercase filenames (for consistency, and because URLs are case-sensitive)

Reply:
Also looks like `hub-edit-token.png` is no longer used now, so we should remove that image.

Reply:
@thaJeztah This image hub-edit-token.png is still applicable. Updated in next commit.